tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.ts

```diff
@@ -314,25 +334,12 @@ export class TfScalarCard extends PolymerElement {
       runs.push(run);
     }
 
-    // Request at most this many runs at once.
-    //
-    // Back-of-the-envelope math: each scalar datum JSON value contains
-    // two floats and a small-ish integer. Floats are about 18 bytes,
-    // since f64s have -log_10(2^-53) ~= 16 digits of precision plus
-    // decimal point and leading zero. Small-ish integers (steps) are
-    // about 5 bytes. Add JSON overhead `[,,],` and you're looking at
-    // about 48 bytes per datum. With standard downsampling of
-    // 1000 points per time series, expect ~50 KB of response payload
-    // per requested time series.
-    //
-    // Requesting 64 time series warrants a ~3 MB response, which seems
-    // reasonable.
-    const BATCH_SIZE = 64;
+    const batchSize = this.batchSize || DEFAULT_BATCH_SIZE;
```

Comment:
I think it is more correct to write `this.batchSize ?? DEFAULT_BATCH_SIZE` since otherwise,
- if `this.batchSize` = 0, batchSize = DEFAULT_BATCH_SIZE
- if `this.batchSize` = -1, batchSize = -1
- if `this.batchSize` = 1, batchSize = 1

Reply:
Yes. We could consider also enforcing that the batchSize > 0 but since this is for internal use only I'll wait to see if anybody ever complains.